lib/src/index.ts

```diff
@@ -14,15 +14,15 @@ export * from './DateRangePicker/DateRangePicker';
 
 export { DateRangePickerToolbar } from './DateRangePicker/DateRangePickerToolbar';
 
-export { Calendar as PickersCalendar } from './views/Calendar/Calendar';
+export { default as PickersCalendar } from './views/Calendar/Calendar';
 
-export { CalendarView as PickersCalendarView } from './views/Calendar/CalendarView';
+export { default as PickersCalendarView } from './views/Calendar/CalendarView';
 
-export { Day as PickersDay } from './views/Calendar/Day';
+export { default as PickersDay } from './views/Calendar/Day';
 
-export { ClockView as PickersClockView } from './views/Clock/ClockView';
+export { default as PickersClockView } from './views/Clock/ClockView';
 
-export { Clock as PickersView } from './views/Clock/Clock';
+export { default as PickersClock } from './views/Clock/Clock';
```

Comment:
This type of change looks weird. Probably current variant is not ideal as weel – we need to just reexport the right component names

Reply:
I think that the correct solution is to match https://github.com/mui-org/material-ui/blob/833861f8f47b77cbb033562f60354bc36d246f37/packages/material-ui/src/index.js#L15-L16:

```jsx
export { default as PickersClock } from './PickersClock';
export * from './PickersClock';
```

But that's for later.